src/wwwroot/controllers/ReportsCtrl.js

```diff
@@ -18,14 +18,30 @@
   ];
 
   function ReportsCtrl($scope, reports, auth, $location, constants, $translate, ModalService, $rootScope, moment) {
+    loadRateLimits();
     applyFilter();
     $scope.moreThanOneDay = false;
     $scope.canSearch = true;
     $scope.searchInProgress = false;
     $scope.curSection = $location.path().substring(1);
     $scope.deliveriesSummary = {};
     $scope.eventsSummary = {};
-
+    $scope.noLimits = true;
+
+    function loadRateLimits() {
+      reports.getStatusPlanLimits()
```

Comment:
You can prepend this line with return in order to allow _a potential client_ to consume the promise.